### DISCUSSION TITLE
aptio setup utility 2020 version 2.19.1268 where lan boot option

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#404] [General]
Hello!

I did buy Mini PC that came with aptio setup utilitty. (American Megatrends BIOS)
Year: 2020
version: 2.19.1268

Where can I find network / lan / ipxe boot mode? I did try all mode boot to change or find these options.
Someone can Help please?

Thank you!

**Comment by doguibnu** [comment]
There are some screenshots to see from bios model
I enter in all setup
I changed to CSM mode, security boot off. in Bios, set enable for Network Stack driver support.
Nothing change or show in First Boot - Lan boot, ipxe boot, network boot. 
I cannot believe that this Bios has not network lan boot mode.


![01-main1-02-15 at 14 08 07](https://user-images.githubusercontent.com/38897311/107977411-d4744080-6f99-11eb-91e7-1934a6467dd5.jpeg)
![02-advanced-2021-02-15 at 14 08 08](https://user-images.githubusercontent.com/38897311/107977490-f5d52c80-6f99-11eb-97b7-847371be33bf.jpeg)
![03-chipset2021-02-15 at 14 08 08](https://user-images.githubusercontent.com/38897311/107977552-0edddd80-6f9a-11eb-8619-28ffb53f94f0.jpeg)
![04-bios2021-02-15 at 14 08 09](https://user-images.githubusercontent.com/38897311/107977617-2a48e880-6f9a-11eb-86e7-7e5ccce7c916.jpeg)
![05-security2021-02-15 at 14 12 39](https://user-images.githubusercontent.com/38897311/107977682-49477a80-6f9a-11eb-97b6-bc81fb9320ba.jpeg)
![06-save and exit2021-02-15 at 14 13 03](https://user-images.githubusercontent.com/38897311/107977777-7300a180-6f9a-11eb-817e-3b0088644541.jpeg)

**Comment by alkisg** [comment]
Hi Douglas, try to disable fast boot and upload screenshots of north/south bridge screens.

**Reply by doguibnu** [reply]
Hi @alkisg 

Here the pcitures that you request
![14-fast-boot2021-02-15 at 15 46 50](https://user-images.githubusercontent.com/38897311/107984044-c9271200-6fa5-11eb-9959-3f1ecff17276.jpeg)
![15-noth-bridge2021-02-15 at 15 45 15](https://user-images.githubusercontent.com/38897311/107984058-ce845c80-6fa5-11eb-8ebc-021cf23155af.jpeg)
![16-north-bridge-openImage 2021-02-15 at 15 45 26](https://user-images.githubusercontent.com/38897311/107984065-d3491080-6fa5-11eb-9044-49ae4232ebca.jpeg)
![17-soth-bridgemage 2021-02-15 at 15 45 35](https://user-images.githubusercontent.com/38897311/107984077-d8a65b00-6fa5-11eb-8202-3ccacbf772a2.jpeg)


Thanks

**Comment by alkisg** [comment]
It's not there either, more screenshots are needed.
If you press enter on `Boot Option #1: [Disabled]`, what options does it have there?
Is there a key like F12 that invokes the BIOS/UEFI boot manager?
Btw you can also ask in https://github.com/ipxe/ipxe/discussions in case anyone there has seen that UEFI firmware.

**Reply by doguibnu** [reply]
Hello!

@alkisg when enter in Boot Option #1 or #2 there are only 3 options:
UEFI boot manager
Windows boot manager
Disable

I am trying other option reading PDF about similar Mobos. Some Mobos show to Enable - Network Stack - save - reset - back to bios and see if network boot modes will be open to change it. But, not! It is not show the lan / network / ipxe. 

About F12 key still not see Bios/UEFI. I will try it. 

Right, I will ask on IPXE github discussions

Thank you
